src/Analyzer.PowerShellRuleEngine/PowerShellRuleEvaluation.cs

```diff
@@ -23,6 +23,9 @@ public class PowerShellRuleEvaluation : IEvaluation
         /// <inheritdoc/>
         public string HelpUri { get; }
 
+        /// <inheritdoc/>
+        public Severity Severity { get; }
```

Comment:
Since this is never set (because TTK doesn't have any severity assignments), shall we assign this an explicit default value?  Say Medium?
``` suggestion
        public Severity Severity { get; } = Severity.Medium
```